model/cpython/metrics.yaml

```
@@ -0,0 +1,39 @@
+groups:
+  - id: metric.cpython.gc.collections
+    type: metric
+    metric_name: cpython.gc.collections
+    brief: "The number of times a generation was collected since interpreter start."
+    note: >
+      This metric reports data from [`gc.stats()`](https://docs.python.org/3/library/gc.html#gc.get_stats)
+    instrument: counter
+    unit: "{collection}"
+    stability: development
+    attributes:
+      - ref: cpython.gc.generation
+        requirement_level: required
+
+  - id: metric.cpython.gc.collected
+    type: metric
+    metric_name: cpython.gc.collected
```

With `cpython.gc.collected` name it can be confusing to have number of object and the amount of memory (if we ever report it).

It'd be great to be specific in the metric name, e.g. call it 
```suggestion
    metric_name: cpython.gc.collected.objects
```

---

I can do that but we don't have any stats in `bytes`, this 3 metrics per generation are everything we have from `gc.get_stats()`

---

it's still better to be explicit. In case new data is exposed in the future, we will be able to provide a meaningful name. With `cpython.gc.collected` we would have a hard time adding memory metrics if they are ever reported by this or any other API.

---

Done, thanks